front-end/.gitignore

```diff
@@ -19,6 +19,8 @@
 npm-debug.log*
 yarn-debug.log*
 yarn-error.log*
+yarn-lock
+package-lock.json
```

Comment:
package-lock.json mustn't be excluded from gitindex, it represent exact dependencies tree structure.

Reply:
But another question that you have yarn.lock so be consitent, leave only one 
for example yarn.lock as you originally use yarn.
+package-lock.json is need to be excluded from git index and repository then, currently it's in index so it doesn't matter for git that you mentioned file in gitignore.